doc/build-unix.md

```diff
@@ -328,8 +328,11 @@ Then build using:
     ./autogen.sh
     ./configure --disable-wallet # OR
     ./configure BDB_CFLAGS="-I${BDB_PREFIX}/include" BDB_LIBS="-L${BDB_PREFIX}/lib -ldb_cxx"
+    
     gmake
 
+*Note* that as of Bitcoin Core version 0.16.0, you need to add --disable-asm at the configure step to disable defailt ASM optimization to compile for FreeBSD. See [here](https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=226155) for details.
```

Comment:
Nit: Use \`--disable-asm\` to get correct formatting :-)

Reply:
Done.